ui/src/app/shared/workflow-template/apply-form/workflow-template.apply-form.component.ts

```diff
@@ -170,6 +170,16 @@ export class WorkflowTemplateApplyFormComponent {
         }
     }
 
+    filterRepo(options: Array<string>, query: string): Array<Repository> | false {
+        if (!options) {
+            return false;
+        }
+        if (!query || query.length < 3) {
+            return options.slice(0, 100);
+        }
+        return options.filter(name => name.toLowerCase().indexOf(query.toLowerCase()) !== -1);
```

Comment:
create a variable above with `let queryLowerCase = query.toLowerCase()` to execute just one time

Reply:
done